WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -350,13 +354,12 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
-+ (XCUICoordinate *)getGestureCoordinate:(FBRouteRequest *)request
++ (XCUICoordinate *)gestureCoordinateWithCoordinate:(CGPoint)coordinate application:(XCUIApplication *)application
 {
-  FBSession *session = request.session;
-  CGVector point = CGVectorMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
-  XCUICoordinate *appCoordinate = [[XCUICoordinate alloc] initWithElement:session.application normalizedOffset:CGVectorMake(0, 0)];
-  XCUICoordinate *gestureCoordinate = [[XCUICoordinate alloc] initWithCoordinate:appCoordinate pointsOffset:point];
-  return gestureCoordinate;
+  // TODO: handle this properly after XCTest starts to respect landscape interface orientation
+  CGPoint point = FBInvertPointForApplication(coordinate, application.frame.size, application.interfaceOrientation);
```

Comment:
Let's make it simple. Most of users don't even use 20% of features we have here. I could think about improving this if there are enough complains in Appium forum. But for now it might take a lot of time to just create a workaround for the stuff, which is not widely used. And time is money 💯